### PR TITLE
create topology doc: fix controller install links

### PR DESCRIPTION
**Pull request by defo89**

### CODE REVIEW

**Review by marcushines — APPROVED** [review]

File: docs/create_topology.md [hunk 1]
```diff
@@ -206,11 +206,8 @@ See more on the
 
 > IMPORTANT: Contact Arista to get access to the cEOS container image.
```
Comment (alexmasi) [inline docs/create_topology.md:207]: can you remove this line?

File: docs/create_topology.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ To manually apply the controller run the following command after cloning the
 Keysight controller yaml:
```
Comment (alexmasi) [inline docs/create_topology.md:186]: and this line

**Review by alexmasi — APPROVED** [review]
Thanks for the fix

File: docs/create_topology.md (hunk 2 above)
Comment (defo89) [reply docs/create_topology.md]: sure, also removed the notice since ixiatg operator images are publicly available

**Review by alexmasi — APPROVED** [review]

**Pull request merged**